src/Amalgam/entity/EntityQueryBuilder.h

```diff
@@ -39,6 +39,18 @@ namespace EntityQueryBuilder
 			|| type == ENT_COMPUTE_ENTITY_KL_DIVERGENCES);
 	}
 
+	//populates deviation data for feature_params from deviation_node
+	inline void PopulateFeatureDeviationData(GeneralizedDistance::FeatureParams &feature_params, EvaluableNode *deviation_node)
```

Comment:
If you are only altering the 'deviation' member of feature_params here, any reason to not just pass that member in instead of whole thing? It'll most likely inlined so probably doesn't matter too much. Unless having the feature_param is good for the context. In which case, looks good.